.github/workflows/ci.yml

```diff
@@ -15,18 +15,19 @@ jobs:
       matrix:
         version: [5.1.5, 5.2.4, 5.3.6, 5.4.7]
 
+    env:
+      LUA_VERSION: ${{ matrix.version }}
+    
```

Comment:
Just to confirm: This is an unrelated refactoring that has no effect on libreadline availability, right?

Reply:
this is unrelated to ubuntu CI. Just updating links.